packages/Modal/stories/Modal.examples.stories.js

```diff
@@ -0,0 +1,208 @@
+import React from "react";
+import { repeat } from "storybook/assets/styles/common.styles";
+import ExampleStory from "storybook/components/ExampleStory";
+import { exampleStoryParameters } from "storybook/assets/storyParameters";
+import { getStoryName } from "storybook/storyTree";
+import styled from "styled-components";
+import tokens from "@paprika/tokens";
+import Button from "@paprika/button";
+import stylers from "@paprika/stylers";
+import ModalStory from "./ModalStory";
+
+import Modal from "../src";
+
+const storyName = getStoryName("Modal");
+
+export default {
+  title: `${storyName}/Examples`,
+  component: Modal,
+};
+
+const DemoFullWidthContent = styled.div`
+  background-image: linear-gradient(to bottom, #79ff85, #70b3ff);
+  height: 1000px;
+`;
+
+export const basic = () => (
+  <ExampleStory component="Modal" storyName="Basic" fileName="./ModalStory.js">
+    <ModalStory>
+      <Modal.Content>
+        {repeat(25, key => (
+          <p key={key}>Some content here</p>
+        ))}
+      </Modal.Content>
+    </ModalStory>
+  </ExampleStory>
+);
+
+basic.story = exampleStoryParameters;
```

Comment:
This doesn't break, but doesn't work as intended, unless the `exampleStoryParameters` are applied to `basic.story.parameters`.  Like this:

https://github.com/acl-services/paprika/pull/1216/files#diff-ee0bffb54044d1dae68748c994226281a259cb7613594c6b61e6a1d0413442efR23